TRIAGING.md

```diff
@@ -0,0 +1,59 @@
+
+### Which triage meeting should I attend?
+
+You should attend the meeting for the repo where your issue was opened, or the "catch all" if your issue has not seen much traction for an extended period of time.
+
+### Do I need to attend for my issue to be addressed/triaged?
+
+Attendance is not required for your issue to be triaged or addressed. All new issues are triaged weekly.
+
+### What happens if my issue does not get covered this time?
+
+During each meeting we seek to address all new issues. However, should we run out of time before your issue is discussed, you are always welcome to attend the next meeting or to follow up on the issue post itself.
+
+### How do I join triage?
+
+Meetings are hosted regularly and can be joined via the links posted on the [OpenSearch Meetup Group](https://www.meetup.com/opensearch/events/) list of events. The events are typically titled `Development Backlog & Triage Meeting`.
+
+After joining the Zoom meeting, you can enable your video/voice to join the discussion. If you do not have a webcam or microphone available, you can still join in via the text chat.
+
+If you have an issue you'd like to bring forth please consider getting a link to the issue so it can be presented to everyone in the meeting.
+
+### Is there an agenda for each week?
+
+Meetings are typically 60 minutes and structured as follows:
+
+1. Initial Gathering: As we gather, feel free to turn on video and engage in informal and open-to-all conversation. After a bit a volunteer will share their screen and proceed with the agenda.
+2. Announcements: If there are any announcements to be made they will happen at the start of the meeting.
+3. Review of New Issues: The meetings always start with reviewing all untriaged issues. For example, the "catch all" meeting reviews [untriaged issues](https://github.com/search?q=label%3Auntriaged+is%3Aopen++org%3Aopensearch-project&type=issues&ref=advsearch&s=created&o=asc), oldest first.
+4. Member Requests: Opportunity for any meeting member to ask for consideration of an issue or pull request.
+5. Untriaged Items: Review any issues that might have had the `untriaged` label removed, but require additional triage discussion.
+6. Pull Request Discussion: Review the status of outstanding pull requests if there's time.
+7. Open Discussion: Allow for members of the meeting to surface any topics without issues filed or pull request created.
+
+There is no specific ordering within each category.
+
+If you have an issue you would like to discuss but do not have the ability to attend the entire meeting, please attend when is best for you and signal that you have an issue to discuss when you arrive.
```

Comment:
Lets remove this line - it invites members to hijack the meeting.  Nothing prevents a member from making this kind of request during the meeting, but I would prefer not writing it down.